Dockerfile

```diff
@@ -50,8 +50,7 @@ RUN cp -rf  ${BASE_PATH}/tomcat/webapps/pentaho/mantle/home/properties/ ${BASE_P
     chmod -R 777  /home/pentaho/* && \
     chmod -R 777  /home/pentaho/	
 USER pentaho
-
+ENV PATH=${BASE_PATH}:/opt/solr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Does the `/opt/solr/bin` directory exists in the container? I guess solr is not installed in the container, is it?